x-pack/plugins/fleet/public/types/ui_extensions.ts

```diff
@@ -136,13 +136,7 @@ export interface PackagePolicyCreateExtension {
  * UI Component Extension is used on the pages displaying the ability to Create a multi step
  * Integration Policy
  */
-export type PackagePolicyCreateMultiStepExtensionComponent =
-  ComponentType<PackagePolicyCreateMultiStepExtensionComponentProps>;
-
-export interface PackagePolicyCreateMultiStepExtensionComponentProps {
-  /** The integration policy being created */
-  newPolicy: NewPackagePolicy;
-}
+export type PackagePolicyCreateMultiStepExtensionComponent = ComponentType<{}>;
```

Comment:
I would not remove these props as could be used in the future (or might be used for other integrations) when we add any kind of integration modification action.

Reply:
As we discussed, I'd leave them removed to avoid unneeded complexity, and put them back when they are really needed.